### PR TITLE
created `stdizer` function - all tests pass

### PR DESCRIPTION
**Pull request by smwatts**
created the stdizer function

Not sure how to run the testing script and to create the Value/Type Error exceptions, please help!

### CODE REVIEW

**Comment by jchen9314** [comment]
@smwatts 
To create the Value/Type Error exceptions in function, 
try the following code:
```
if not isinstance(X, (pd.DataFrame, np.ndarray)):
     raise TypeError(error messsage)
```
Hope it would help you.

**Comment by jchen9314** [comment]
For running the testing script, I just use `pytest test_file_name` in the command line and see if all test cases pass.

**Comment by smwatts** [comment]
Thanks @jchen9314!

File: tests/test_stdizer.py [hunk 1]
```diff
@@ -53,6 +53,8 @@ def correct_argument_values():
...
     np_data_2 = np.asarray(data_2)
     with pytest.raises(ValueError):
-        pre.stdizer(np_data, method="not a valid method")
-        pre.stdizer(np_data, method="own", method_args=[[1],[1,2]])
-        pre.stdizer(np_data, method="own", method_args=[[1,1],[1,0]])
\ No newline at end of file
+        std.stdizer(np_data, method="not a valid method")
+        std.stdizer(np_data, method="own", method_args=[[1],[1,2]])
+        std.stdizer(np_data, method="own", method_args=[[1,1],[1,0]])
+    with pytest.raises(ValueError):
```
Comment (jchen9314) [inline tests/test_stdizer.py]: 1. There are duplicated code in line 59 and 60. 
2. Did you use `data_2` and `np_data_2` in this test case?
3. To convert the data type, we could use `np.array()`.

**Review by jchen9314 — CHANGES_REQUESTED** [review]
Good job! Few comments on test_stdizer.py.

File: tests/test_stdizer.py (hunk 1 above)
Comment (smwatts) [reply tests/test_stdizer.py]: fixed! good catch thanks

**Review by jchen9314 — APPROVED** [review]

**Review by apchiodo — APPROVED** [review]

**Pull request merged**